inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/resource/sink/ck/ClickHouseSqlBuilder.java

```diff
@@ -66,7 +67,10 @@ public static String buildCreateTableSql(ClickHouseTableInfo table) {
         }
         // Construct columns and partition columns
         sql.append(buildCreateColumnsSql(table.getFieldInfoList()));
-        if (StringUtils.isNotEmpty(table.getEngine())) {
+        if (StringUtils.isNotBlank(table.getEngine()) && Objects.equals("ReplicatedMergeTree", table.getEngine())) {
+            sql.append(
+                    " ENGINE = ReplicatedMergeTree('/clickhouse/tables/{shard}/{database}/{table}/data', '{replica}')");
```

Comment:
Is there only one engine? 
I think it's better to make an engine emun, instead of making branch for each engine type.

Reply:
This only `ReplicatedMergeTree`.